tests/intern-browserstack.ts

```diff
@@ -0,0 +1,14 @@
+export * from './intern';
```

Comment:
Note that Intern no longer uses JS modules for config. The configuration is now a [JSON file](https://github.com/theintern/intern/blob/master/docs/configuration.md#config-file).